uc_attribute/views/uc_attribute.views.inc

```diff
@@ -0,0 +1,192 @@
+    $data['uc_product_options_' . $row->aid . '_' . $row->oid]['table']['join']['node'] = array(
+      'table' => 'uc_product_options',
+      'left_field' => 'nid',
+      'field' => 'nid',
+      'extra' => array(
+        array(
+          'field' => 'oid',
+          'value' => $row->oid,
+        ),
+      ),
+    );
+    $data['uc_product_options_' . $row->aid . '_' . $row->oid]['table']['group'] = t('Product attributes');
+    $data['uc_product_options_' . $row->aid . '_' . $row->oid]['price'] = array(
+      'title' => t('Price for !aid/!oid', array('!aid' => $row->label ? $row->label : $row->attr_name, '!oid' => $row->option_name)),
+      'help' => 'Price of the product with the selected option.',
+      'field' => array(
+        'handler' => 'uc_product_handler_field_price',
+        'click sortable' => TRUE,
+        'float' => TRUE,
+      ),
+    );
+    $data['uc_product_options_' . $row->aid . '_' . $row->oid]['cost'] = array(
+      'title' => t('Cost for !aid/!oid', array('!aid' => $row->label ? $row->label : $row->attr_name, '!oid' => $row->option_name)),
+      'help' => 'Cost of the product with the selected option.',
+      'field' => array(
+        'handler' => 'uc_product_handler_field_price',
+        'click sortable' => TRUE,
+        'float' => TRUE,
+      ),
+    );
+    $data['uc_product_options_' . $row->aid . '_' . $row->oid]['weight'] = array(
+      'title' => t('Weight for !aid/!oid', array('!aid' => $row->label ? $row->label : $row->attr_name, '!oid' => $row->option_name)),
+      'help' => 'Weight of the product with the selected option.',
+      'field' => array(
+        'additional fields' => array(
+          'weight_units' => array(
+            'table' => 'uc_products',
+            'field' => 'weight_units',
+          ),
+        ),
+        'handler' => 'uc_product_handler_field_weight',
+        'click sortable' => TRUE,
+        'float' => TRUE,
+      ),
+    );
+  }
+
+  // Create a filter for each product attribute.
+  $result = db_query("SELECT aid, name, description FROM {uc_attributes}");
+  foreach ($result as $row) {
+
+    $data['uc_order_products']['attr_' . $row->aid] = array(
+      'title' => $row->name,
+      'group' => t('Ordered product: Attributes'),
+      'help' => 'Filter ordered products by this attribute.',
+      'filter' => array(
+        'handler' => 'UcAttributeHandlerFilterAttr',
+      ),
+      'aid' => $row->aid,
+    );
+
+    // Create a filter for each product attribute.
+    $data['uc_product_adjustments']['pattr_' . $row->aid] = array(
+      'title' => $row->name,
+      'help' => 'Filter by this attribute if it has its own SKU on the products\'s adjustments.',
+      'filter' => array(
+        'handler' => 'UcAttributeHandlerFilterProductAttr',
+      ),
+    );
+  }
+
+  $data['uc_product_adjustments']['table']['group'] = t('Product adjustments');
+
+  $uc_product_adjustments = drupal_get_schema_unprocessed('uc_attribute', 'uc_product_adjustments');
```

Comment:
This should be backdrop_get_schema_unprocessed

Reply:
Yes, fixed, thanks!